### PR TITLE
ADD example script to calc dynapt in random machines with kickmaps

### PR DESCRIPTION
**Pull request by Gabrielrezende-asc**

### CODE REVIEW

File: examples/calc_dynapt_ids.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    random_machines.seed = seed
+
+    random_machines.configure_corrections()
+
+    nr_steps = 5
+
+    deltadata = DELTA52Data()
+
+    filenames = [
+
```
Comment (fernandohds564) [inline examples/calc_dynapt_ids.py:80]: this empty line here is weird, isn't it?

File: examples/calc_dynapt_ids.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        deltadata.get_kickmap_filename(meas_flag=True, dp=0, dgv=0),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=0, dgv=13.125),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=0, dgv=26.25),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=-13.125, dgv=13.125),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=-13.125, dgv=26.25),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=-26.25, dgv=13.125),
+        deltadata.get_kickmap_filename(meas_flag=True, dp=-26.25, dgv=26.25)
+
```
Comment (fernandohds564) [inline examples/calc_dynapt_ids.py:88]: this one too.

**Review by fernandohds564 — APPROVED** [review]

**Comment by murilobalves** [comment]
There is a `dynap` module in apsuite, maybe we should keep this naming in your script, `calc_dynap_ids.py` then.

**Review by murilobalves — APPROVED** [review]

**Comment by Gabrielrezende-asc** [comment]
> There is a `dynap` module in apsuite, maybe we should keep this naming in your script, `calc_dynap_ids.py` then.

Renamed

**Review by xresende — APPROVED** [review]

**Pull request merged**